src/main/resources/assets/crossroads/lang/en_us.json

```diff
@@ -307,7 +307,7 @@
 
 	"item.minecraft.potion.effect.sedation": "Potion of Sedation",
 	"item.minecraft.splash_potion.effect.sedation": "Splash Potion of Sedation",
-	"item.minecraft.lingering_potion.effect.sedation": "Lingering Potion of Sedation",
+	"item.minecraft.lingering_potion.`effect.sedation": "Lingering Potion of Sedation",
```

Comment:
Why the added backtick?

Reply:
Yep, that was a mistake. Probably happened during an alt+tab. Sorry.

Fixed.